pilot/pkg/xds/xds_test.go

```diff
@@ -300,26 +301,49 @@ func TestSidecarListeners(t *testing.T) {
 
 func TestMeshNetworking(t *testing.T) {
 	s := NewFakeDiscoveryServer(t, FakeOptions{
-		ConfigString: `
```

Comment:
is there value in both tests?  Like should we add your new one instead of replacing?